src/APIM_ARMTemplate/apimtemplate/Extractor/Models/ExtractorParameters.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace apimtemplate.Extractor.Models
+{
+    public class ExtractorParameters
+    {
+        public string sourceApimName { get; private set; }
```

Comment:
Are we using `sourceApimName` because of serialization or another reason? Because we should use `SourceApimName`.

Reply:
If serialization, we should use attributes to change the casing during serialization IMO, but follow-up PR for this is preferred

Reply:
Yes, I wanted to test the API's and change the property naming. I wanted to follow this PR with such a change

Reply:
will do it in #617